src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todo: Todo;
+  user: User | null;
+  isLoading: boolean;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  todo,
+  user,
+  isLoading,
+  onClose,
+}) => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {isLoading && !user ? (
```

Comment:
```suggestion
      {isLoading ? (
```
You should show the loader only if you are loading something